docs/config/README.md

```diff
@@ -52,56 +52,11 @@ You can customize which network is used by default when running Hardhat by setti
 
 ### Hardhat Network
 
-Hardhat comes built-in with a special network called `hardhat`. When using this network, an instance of the [Hardhat Network](../hardhat-network/README.md) will be automatically created when you run a task, script or test your smart contracts
+Hardhat comes built-in with a special network called `hardhat`. When using this network, an instance of the [Hardhat Network](../hardhat-network/README.md) will be automatically created when you run a task, script or test your smart contracts.
 
 Hardhat Network has first-class support of Solidity. It always knows which smart contracts are being run and exactly what they do and why they fail. Learn more about it [here](../hardhat-network/README.md).
```

Comment:
This is not part of this PR, but worth mentioning that this paragraph is misleading. It doesn't always know which smart contracts are being run, nor why they fail. The idea that is a best-effort thing should be made explicit here.